CONTRIBUTING.md

```diff
@@ -2,58 +2,57 @@
 
 This page provides information about contributing code to the Jenkins core codebase.
 
-:exclamation: There's a lot more to the Jenkins project than just code. For information on contributing to the Jenkins project overall, check out [Participate].
+:exclamation: There's a lot more to the Jenkins project than just code. For information on contributing to the Jenkins project overall, here's a page on how to [Participate].
```

Comment:
'For information on contributing to the Jenkins project overall, here's a page on how to [Participate].'  - **suggestion**: For more information on the ways that you can contribute to the Jenkins project, see [Participate].

Reply:
```suggestion
:exclamation: There's a lot more to the Jenkins project than just code. For more information on the ways that you can contribute to the Jenkins project, see [Participate].
```

@andreashouben GitHub suggestions are quite handy when it comes to reviews and suggestions